mtl/multitask_tutorial.py

```diff
@@ -204,7 +204,11 @@ def split_data(data, labels, seed=123):
 # Note that `Task` objects are not dependent on a particular dataset; multiple datasets can be passed through the same modules for pre-training or co-training.
 
 # %% [markdown]
-# We'll now define the square task, but more succinctly—for example, using the fact that the default name for an `Operation` is its `module_name` (since most tasks only use their modules once per forward pass). We'll also define the square task to share the input module with the circle task to demonstrate how to share modules. (Note that this is purely for illustrative purposes; for this toy task, it is very possible that this is not the optimal arrangement of modules).
+# We'll now define the square task, but more succinctly—for example, using the fact that the default name for an `Operation` is its `module_name` (since most tasks only use their modules once per forward pass).
```

Comment:
nit: maybe split up the comments and steps with interleaved code if it still flows?

Reply:
Actually, the point of this section is do it all in one line. We built the previous task (circle_task) piece by piece, one chunk at a time. But now we want to show that if you know what you're doing, it can be defined all at once.